GitUpdateScript.sh

```diff
@@ -94,26 +94,30 @@ rm Temp_dir_log #rm removes/dels the file
 }
 
 home_dir=$(pwd)  #v1 holds the home dir
-echo "Git Repositories Found:"
+
 
 # todo Count from find instead of iterating loop from file
-find -name .git| tee Temp_dir_log  #finds file with name .gitignore and stores results in file called FILENAME
 
-echo "Going to each git repositories and update:"
-while read file_dir
-do
 
-	GoToGitRepositoryDirectory
+arrayFileDir=($(find -name .git))
+numberOfGitRepos=$(find -name .git|wc -l)  #wc is word count -l makes it count lines, | is pipe and find finds all .git
+
+
+echo "Going to each git repository and updating:"
+
+for (( n=0; n<$numberOfGitRepos; n++ ))
```

Comment:
do you need double parenthesis here? ((

Reply:
Double brackets are for arithmetic operations. You can omit dollar signs from your variables for readability. `$n++` becomes `n++`.